mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -178,12 +208,21 @@ CLASS none/se net/minecraft/entity/Entity
 		ARG 0 value
 	METHOD h getSquaredDistanceTo (Lnone/se;)D
 		ARG 0 entity
+	METHOD h setInvisible (Z)V
+		ARG 0 value
 	METHOD h_ getName ()Ljava/lang/String;
+	METHOD i initDataTracker ()V
+	METHOD i setOnFire (I)V
+		ARG 0 seconds
+	METHOD i setInvulnurable (Z)V
+		ARG 0 value
 	METHOD i_ getNameComponent ()Lnone/fa;
 	METHOD j setCustomNameVisible (Z)V
 		ARG 0 value
-	METHOD k getTrackedBoolean (I)Z
-		ARG 0 id
+	METHOD k getEntityFlag (I)Z
+		ARG 0 index
+	METHOD l setAir (I)V
```

Comment:
I've been using the suffix `Timer` for anything that counts down in ticks. How about that?

Also, go with `Air` or maybe `Breath`?